wordpress-mobile/WordPress-iOS
# Media Editing: Update pod to add drawing capability

This PR pulls in the latest MediaEditor pod version to allow drawing on images in the editor (refs https://github.com/wordpress-mobile/MediaEditor-iOS/pull/19). 

**To test:**

* Build and run on iOS 13. Do a smoke test of the drawing functionality detailed in the PR referenced above.
* Build and run on iOS 12. Ensure that the drawing tool does not appear.

PR submission checklist:
